src/application.ts

```diff
@@ -32,11 +32,16 @@ export class HelloWorldApp extends RepositoryMixin(Application) {
     server
       .bind(AuthenticationBindings.STRATEGY)
       .toProvider(MyAuthStrategyProvider);
-    await super.start();
-    let port = await server.get(RestBindings.PORT);
-    console.log(`HTTP server listening on port ${port}`);
-    console.log(
-      `Run \'curl localhost:${port}/helloworld?name=YOUR_NAME\' to try it out`,
-    );
+    try {
+      await super.start();
+      let port = await server.get(RestBindings.PORT);
+      console.log(`HTTP server listening on port ${port}`);
+      console.log(
+        `Run \'curl localhost:${port}/helloworld?name=YOUR_NAME\' to try it out`,
+      );
+    } catch (err) {
+      console.error('Cannot start the application! ', err);
+      process.exit(1);
+    }
```

Comment:
Please revert changes in this file per my comment above.

You don't want `app.start()` to swallow errors (or convert them to a console log or a forceful process exit), because when you call `app.start()` from a test and the start fails, then you want the test to fail too, and let the test runner to report the test failure in a usual way.